egov-dss-dashboards/dashboard-analytics/MasterDashboardConfig.json

```diff
@@ -11975,6 +11975,54 @@
           ]
         }
       ]
+    },
+    {
+      "name" : "DSS_HEALTH_IRS_TEAM_LEAD_DASHBOARD",
+      "id" : "irs-team-lead-dashboard",
+      "isActive" : true,
```

Comment:
Keep the chart inactive, DSS Dashboard is not enabled in React app yet,  we will test it in mobile